CryptoPrices/Sources/MyCoin/Sources/MyCoin/MyCoinView/MyCoinView.swift

```diff
@@ -12,13 +12,20 @@ public struct MyCoinView: View {
     @EnvironmentObject var myCoinState: MyCoinState
 
     public var body: some View {
-        NavigationView {
-            contentView
-                .navigationBarBackButtonHidden(true)
-                .navigationBarItems(leading: backButton, trailing: likeButton)
-                .navigationTitle("Ethereum") // TODO: Remove dummy
-                .navigationBarTitleDisplayMode(.inline)
-        }
+        contentView
+            .navigationBarBackButtonHidden()
+            .navigationBarItems(leading: backButton, trailing: likeButton)
+            .navigationTitle("Ethereum") // TODO: Remove dummy
+            .navigationBarTitleDisplayMode(.inline)
+            .gesture(
+                DragGesture()
+                    .onEnded { value in
+                        // Handle swipe left-to-right
+                        if value.startLocation.x < 20.0 && value.translation.width > 50.0 {
+                            myCoinState.didSelectBack = true
+                        }
+                    }
```

Comment:
Hmm I don't think there is a requirement for swiping back to go back to the previous screen but yeah it is a nice to have thing 👍 

Just a thought, maybe we can consider creating a view extension for this swipe-back gesture so we can apply it for multiple screens when the project scales up, as of now it is still good to just apply it here locally for `MyCoinView` 🙏

Reply:
Thank you 👍

For now, I think I'd apply locally for `MyCoinView`. Later on we can make it as an extension as your suggestion 🙏
Note that in that case, it would be placed in a separate package, i.e. `Styleguide` in our case.